mcs/class/System.Core/System.Core_test.dll.sources

```diff
@@ -82,3 +82,4 @@ System.Runtime.CompilerServices/DynamicAttributeTest.cs
 System.Linq/ParallelEnumerableTests.cs
 System.Linq/ParallelTestHelper.cs
 System.Threading.Tasks/TaskExtensionsTests.cs
+System.Threading.Tasks/ThreadPrincipalTests.cs
```

Comment:
Why is this test here and not in mscorlib ?

Reply:
Added the reporter's test before I identified the fix. I can move it.

Reply:
This should definitely be in `mscorlib` test suite